bin/main.rb

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/ruby
+#
+require_relative '../lib/tictactoe'
+
+class MainIO
+  def initialize(stdin: '', stdout: [])
+    @stdin = stdin
+    @stdout = stdout
+    @accepted_input = [*1..9].map!(&:to_s)
+  end
+
+  def output_to_user(string)
+    @stdout.puts string
+  end
+
+  def prompt_user_for_position
+    input = $stdin.gets.chomp
+    int_to_return = @accepted_input.include?(input) ? input.to_i - 1 : nil
+    int_to_return 
+  end
+end
+
+class Game
```

Comment:
I like how small this class is! Are there any tests for it?

Reply:
I am not sure `main.rb` is the best location for this.

Reply:
Ditto these comments - it seems like a testable chunk of logic, and I like the game_logic interface (it's nicely descriptive, I feel like it's easy to intuit what's going on). I'd recommend extracting and testing it for the same reasons as `MainIO`